src/schema/devcards.ts

```diff
@@ -172,13 +190,20 @@ export const resolvers: IResolvers<any, Context> = {
 
       // Avoid caching issues when devcard is generated again
       const randomStr = Math.random().toString(36).substring(2, 5);
-      const queryStr = qs.stringify({
-        type,
-        r: randomStr,
-      });
+      const queryStr = new URLSearchParams(
```

Comment:
We can just use the plain URL one, no need to use the lodash ones I think.

Reply:
If we do not add the `omitBy`, we end up with `type=undefined` in the query string. I'd rather not have empty parameters there :)